doc/upgrade-cluster.md

```diff
@@ -221,6 +221,32 @@ INFO[5930] error syncing cluster version, upgrade failed. failed to sync bookkee
 ...
 ```
 
+### Handling Upgrade from one version to another
```

Comment:
Lets add this to existing section titled "Trigger an upgrade" instead of creating a new section.